domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/mapper/FieldTypeListItemMapper.java

```diff
@@ -8,22 +9,22 @@
 
 public class FieldTypeListItemMapper {
     public List<FixedListItem> entityToModel(final List<FieldTypeListItemEntity>
-                                                                    fieldTypeListItemEntities) {
+                                                 fieldTypeListItemEntities) {
         List<FixedListItem> fixedListItems = new ArrayList<>();
         fieldTypeListItemEntities.stream().forEach(fieldTypeListItemEntity ->
-                                                   {
-                                                       FixedListItem fixedListItem = new FixedListItem();
-                                                       fixedListItem
-                                                           .setCode(fieldTypeListItemEntity.getValue());
-                                                       fixedListItem
-                                                           .setLabel(fieldTypeListItemEntity.getLabel());
-                                                       fixedListItem
-                                                           .setOrder(fieldTypeListItemEntity.getOrder());
-                                                       fixedListItems.add(fixedListItem);
-                                                   }
-                                                  );
-        return fixedListItems;
+            {
+                FixedListItem fixedListItem = new FixedListItem();
+                fixedListItem
+                    .setCode(fieldTypeListItemEntity.getValue());
+                fixedListItem
+                    .setLabel(fieldTypeListItemEntity.getLabel());
+                fixedListItem
+                    .setOrder(fieldTypeListItemEntity.getOrder());
+                fixedListItems.add(fixedListItem);
+            }
+        );
+        return new FieldTypeListItemSorter().sortFixedListItems(fixedListItems);
```

Comment:
this applies to all list types, including multi select. Perhaps we could rename to something more generic like:
`sortListItems()`